Formula/cockroach.rb

```diff
@@ -1,42 +1,43 @@
 class Cockroach < Formula
   desc "Distributed SQL database"
   homepage "https://www.cockroachlabs.com"
-  version "1.0"
+  url "https://binaries.cockroachdb.com/cockroach-v19.1.4.darwin-10.9-amd64.tgz"
+  version "19.1.4"
+  sha256 "6911796049865aa1c27d11904893ea6f7a031bdfe90f531a4ea510fde3568439"
+  # This revision facilitates conversion from homebrew-core to the tap.
+  # TODO(bdarnell): Remove this once we've updated to a version that never appeared in homebrew-core.
   revision 1
-  url "https://binaries.cockroachdb.com/cockroach-v1.0.src.tgz"
-  sha256 "ca87b10eec688195e0df4f85431b019f2980ae4b511ee321f91f945315efeb76"
-  head "https://github.com/cockroachdb/cockroach.git"
+
+  bottle :unneeded
 
   def install
-    raise <<-EOS.undent
-      The CockroachDB formula has moved into Homebrew proper.
-      Please instead run:
+    bin.install "cockroach"
+    system "#{bin}/cockroach", "gen", "man", "--path=#{man1}"
 
-          $ brew uninstall cockroachdb/cockroach/cockroach
-          $ brew untap cockroachdb/cockroach
-          $ brew install cockroach
+    bash_completion.mkpath
+    system "#{bin}/cockroach", "gen", "autocomplete", "bash", "--out=#{bash_completion}/cockroach"
 
-    EOS
+    zsh_completion.mkpath
+    system "#{bin}/cockroach", "gen", "autocomplete", "zsh", "--out=#{zsh_completion}/cockroach"
```

Comment:
Not sure if this matters, but zsh_completion files are typically named with a leading underscore, so `#{zsh_completion}/_cockroach` in this case. Here's a random but representative example: https://github.com/Homebrew/homebrew-core/blob/7ee32113351bbd913b90f9578bcd52dfe85d675e/Formula/kubernetes-cli.rb#L38.

Reply:
Thanks! I'm not a zsh user so I haven't tested this part. The underscore is indeed called out in the docs: http://zsh.sourceforge.net/Doc/Release/Completion-System.html#Initialization